.github/docker/Dockerfile.python

```diff
@@ -1,3 +1,3 @@
-FROM python:3.11.13-alpine3.22
+FROM python:3.11.13-bookworm
```

Comment:
chatted with @pkwarren about switching this from alpine, because debian is what distroless uses as a base.

Reply:
Thanks for making this change. If/when other python plugins are updated, we might need to tweak their Dockerfiles but this will be good to have build and runtime be on the same Linux base.